doc/install.rst

```diff
@@ -0,0 +1,145 @@
+Source
+******************************************************************************
+
+You can build nanodbc library, build and run tests using `CMake`_.
+
+.. code-block:: console
+
+  git clone https://github.com/lexicalunit/nanodbc.git
+  cd nanodbc
+  mkdir build
+  cd build
+  cmake ..
+  cmake --build .
+  ctest -V --output-on-failure
+
+Requirements
+==============================================================================
+
+* C++ compiler with C++11/C++14 support
+* `CMake`_ 3.0.0 or later
+* ODBC SDK (unixODBC, iODBC, Windows SDK)
+
+Optionally, you will also need:
+
+* ODBC drivers, depending on DBMS you want to target (eg. running tests).
+* Boost, `Boost.Locale`_ as alternative for Unicode conversions.
+* `libc++`_, alternative C++11 and later implementation.
+
+Building
+==============================================================================
+
+Although detailed build process depends on CMake generator used,
+number of common targets are always available.
+
+For example, CMake configuration using Makefiles generator:
+
+.. code-block:: console
+
+  cd nanodbc
+  mkdir build
+  cd build
+  cmake -G "Unix Makefiles" [options] ..
+  make          # creates shared library
+  make nanodbc  # creates shared library
+  make tests    # builds the tests
+  make test     # runs the tests
+  make check    # builds and then runs tests
+  make examples # builds all the example programs
+  make install  # installs nanodbc.h and shared library
+
+CMake Options
+------------------------------------------------------------------------------
+
+The following build options are available via CMake command-line
+option ``-D``.
+
+Use the standard CMake option ``-DBUILD_SHARED_LIBS=ON`` to build nanodbc
+as shared library.
+
+List of nanodbc-specific options
+
+* ``NANODBC_DISABLE_ASYNC`` - ``ON|OFF``, default is ``OFF``.
+  Disables all async features. Can resolve build issues in older ODBC versions.
+* ``NANODBC_ENABLE_LIBCXX`` - ``ON|OFF``, default is ``ON``.
+  Enables usage of libc++ if found on the system.
+* ``NANODBC_EXAMPLES`` - ``ON|OFF``, default is ``ON``.
+  Enables building of examples.
+* ``NANODBC_HANDLE_NODATA_BUG`` - ``ON|OFF``, default is ``OFF``.
+  Provided to resolve issue `#33 <https://github.com/lexicalunit/nanodbc/issues/33>`_.
+* ``NANODBC_INSTALL`` - ``ON|OFF``, default is ``ON``.
+  Enables install target.
+* ``NANODBC_ODBC_VERSION`` - ``SQL_OV_ODBC3`` or ``SQL_OV_ODBC3_80``,
+  default is ``SQL_OV_ODBC3_80``, if available.
+  Sets the ODBC version macro for nanodbc to use.
+* ``NANODBC_TEST`` - ``ON|OFF``, default is ``ON``.
+  Enables tests target (alias ``check``).
+* ``NANODBC_USE_BOOST_CONVERT`` - ``ON|OFF``, default is ``OFF``.
+  Provided as workaround to issue `#44 <https://github.com/lexicalunit/nanodbc/issues/44>`_.
+* ``NANODBC_USE_UNICODE`` - ``ON|OFF``, default is ``OFF``.
+  Enables full unicode support. ``nanodbc::string`` becomes ``std::u16string`` or ``std::u32string``.
```

Comment:
@bryevdv I've switched to the definition list in 3acae523119c8b53e2b3dce2097c15b2fbb5373e

Regenerated HTML at http://mateusz.loskot.net/tmp/nanodbc/